tests/testthat/test-5getExperimentCrossAssociation.R

```diff
@@ -3,8 +3,25 @@ context("getExperimentCrossAssociation")
 
 test_that("getExperimentCrossAssociation", {
     
-    # Get data
-    mae <- microbiomeDataSets::peerj32()
+    # Try 5 times to fetch the data
+    for(i in seq_len(5) ){
+        mae <- tryCatch(
+            {
+                # Try to fetch the data 
+                microbiomeDataSets::peerj32()
+            },
+            error = function(cond) {
+                # If it was not possible to fetch the data, give FALSE
+                return(NULL)
+            }
+        )
+        # Break if mae has the data
+        if( !is.null(mae) ){
+            break
+        }
+    }
+    # Run tests if the data fetch was successful
+    if( !is.null(mae) ){
     ############################### Test input ###############################
```

Comment:
Tests used peerj32 data and some test reference values are hard-coded. I changed this little bit. --> it is tested 5 times if the data can be fetched from microbiomeDataSets. If it was successful, tests are run. otherwise they are skipped